src/inference/src/ie_core.cpp

```diff
@@ -851,7 +851,13 @@ class CoreImpl : public ie::ICore, public std::enable_shared_from_this<ie::ICore
                                                 const std::map<std::string, std::string>& config,
                                                 const std::function<void(const CNNNetwork&)>& val = nullptr) override {
         OV_ITT_SCOPE(FIRST_INFERENCE, ie::itt::domains::IE_LT, "Core::LoadNetwork::Path");
-        auto parsed = parseDeviceNameIntoConfig(deviceName, config);
+        // disable auto-batch related when loading from modelPath
+        std::string device = deviceName;
+        std::map<std::string, std::string> config_with_batch = config;
+        CleanUpProperties(device, config_with_batch, ov::auto_batch_timeout);
+        CleanUpProperties(device, config_with_batch, ov::hint::allow_auto_batching);
```

Comment:
Why do we need to add more and more AUTO related things into the ov::Core?
Core should be plugin independent.

Reply:
@ilyachur First, agree with you that core should be plugin independent.

This is not auto plugin related PR, but batch plugin related PR.

This PR is short term w/a for 2022.3 to avoid passing model path to batch plugin. @jane-intel will fix model path handling issue in batch plugin by another PR. After fix PR merged, we can revert this w/a.

Reply:
> Why do we need to add more and more AUTO related things into the ov::Core? Core should be plugin independent.

this is not auto related, if you load model path to GPU, there will also be issues(see 83949).
To be short, we need to decide if batch should be enabled when compile model through model path:
if yes, we need to figure out a way to judge if the network is batchable without reading the model file to cnnNetwork.
if not, or, we prefer the benefit of time saved from read model, then this PR is the right one to go, we don't need batching when loading from model path.

Reply:
> @ilyachur First, agree with you that core should be plugin independent.
> 
> This is not auto plugin related PR, but batch plugin related PR.
> 
> This PR is short term w/a for 2022.3 to avoid passing model path to batch plugin. @jane-intel will fix model path handling issue in batch plugin by another PR. After fix PR merged, we can revert this w/a.

In this case we need to have a mark, that this WA must be removed in next release.



> this is not auto related, if you load model path to GPU, there will also be issues(see 83949).
> To be short, we need to decide if batch should be enabled when compile model through model path:
> if yes, we need to figure out a way to judge if the network is batchable without reading the model file to cnnNetwork.
> if not, or, we prefer the benefit of time saved from read model, then this PR is the right one to go, we don't need batching when loading from model path.

From my point of view, plugins should be responsible for handling different parameters, not core. Can we down this logic to plugins?

Reply:
@ilyachur Agree with you to add a mark here for this WA. Then we can merge this PR to avoid issue and discuss the final solution for next release in meeting.

Reply:
I don't want to merge this PR. From the logs in ticket we see AUTO passes invalid keys to CPU plugin -  why?

Reply:
@ilya-lavrenov  Hi ilya, allow_auto_batching is a key been used and handled in ie core:
https://github.com/openvinotoolkit/openvino/blob/master/src/inference/src/ie_core.cpp#L751
considering below case:
e.g, ie.compile_model("AUTO:CPU,GPU", ov::allow_auto_batching(false))
auto will need to pass through the batch keys for loadnetwork to CPU/GPU, otherwise, the default config for auto batch is true always.

Reply:
I have implement the fix in AUTO for step 1 to resolve the jira and add tests, please review again @ilya-lavrenov